.ci/validation.sh

```diff
@@ -492,7 +492,7 @@ javac8)
   mkdir -p target
   for file in "${files[@]}"
   do
-    javac -d target "${file}"
+    javac --release 8 -d target "${file}"
   done
```

Comment:
Why we need javac8  validation ?

Reply:
Can we remove https://github.com/checkstyle/checkstyle/blob/3a790e87be85f2adfd75fe9e02f16f00deec2f98/.ci/validation.sh#L499

Reply:
> Why we need javac8 validation ?

Some input files are incompatible with Eclipse compiler. These inputs should be compiled with javac11 now.

Reply:
done

Reply:
Yes, all should be jdk11, jdk8 should gone completely